src/test/javascript/spec/component/programming-exercise/programming-exercise-test-case-passed-builds-charts.component.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { ComponentFixture, TestBed, fakeAsync, tick } from '@angular/core/testing';
+import { NgbModule } from '@ng-bootstrap/ng-bootstrap';
+
+import { TestCasePassedBuildsChartComponent } from 'app/exercises/programming/manage/grading/charts/test-case-passed-builds-chart.component';
+import { TestCaseStats } from 'app/entities/programming-exercise-test-case-statistics.model';
+
+describe('TestCasePassedBuildsChartComponent', () => {
+    let comp: TestCasePassedBuildsChartComponent;
+    let fixture: ComponentFixture<TestCasePassedBuildsChartComponent>;
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            imports: [NgbModule],
+            declarations: [TestCasePassedBuildsChartComponent],
```

Comment:
```suggestion
            imports: [],
            declarations: [TestCasePassedBuildsChartComponent, MockDirective(NgbTooltip)],
```

Reply:
Applied together with the other suggestions. :+1: